content/md/en/docs/test/check-runtime.md

```diff
@@ -27,14 +27,14 @@ Prior [`TestExternalities`](https://paritytech.github.io/substrate/master/sp_sta
 for writing unit and integrated tests with mock data, but lacked the ability to test against a chain's actual state.
 The `try-runtime` tool extends `TestExternalities` and `BasicExternalities` by retrieving state using the following RPC endpoints for the node:
 
-- [`get_storage`](https://paritytech.github.io/substrate/master/remote_externalities/trait.RpcApiClient.html#method.get_storage)
-- [`get_keys_paged`](https://paritytech.github.io/substrate/master/remote_externalities/trait.RpcApiClient.html#method.get_keys_paged)
+- [`get_storage`](https://github.com/paritytech/substrate/blob/master/utils/frame/remote-externalities/src/lib.rs#L338)
+- [`get_keys_paged`](https://github.com/paritytech/substrate/blob/master/utils/frame/remote-externalities/src/lib.rs#L364)
```

Comment:
Throughout, calling out line numbers (by element id in the url) on `master` is asking for trouble, as those can be wrong without warning anytime but the link still works.

Reply:
Very good point - I will try to figure out an alternative! 🙏🏽